package.json

```diff
@@ -10,7 +10,9 @@
     "test": "npm run test:functional:start && npm run test:functional && npm run test:functional:stop",
     "test:functional": "node ./test/functional/environment.js",
     "test:functional:start": "node ./test/functional/environment.js --start-environment",
-    "test:functional:stop": "node ./test/functional/environment.js --stop-environment"
+    "test:functional:stop": "node ./test/functional/environment.js --stop-environment",
+    "sonar": "npm run test:coverage && pipedrive-sonar-scanner",
```

Comment:
Could this be separate? so sonar command only runs the scanner?

Reply:
But wouldn't it need the coverage generated by `test:coverage`?

Reply:
where is `sonar` command executed? If it does it in jenkins pipeline, won't it run tests twice in that case?

Reply:
I'm not sure where it's executed, I think pipeline yes. I'll separate these.